src/Directory.Build.targets

```diff
@@ -17,7 +17,10 @@
     <!-- Set output type of test projects to EXE.  This will allow us to run tests on .NET Core via F5 or "dotnet run" -->
     <OutputType>Exe</OutputType>
     <GenerateProgramFile>false</GenerateProgramFile>
-
+    
+    <!-- Makes xunit print all test names in stdout -->
+    <XunitOptions>$(XunitOptions) -diagnostics</XunitOptions>
```

Comment:
Preventing arcade from overriding the xunit json makes this line useless. But I guess it doesn't hurt, in case other random things overwrite the json in the future.